### DISCUSSION TITLE
SyncReturn<Result<MyStructType>>

### DISCUSSION THREAD

**Discussion by talhazengin** [fzyzcjy/flutter_rust_bridge#1088] [Q&A]
I could not use SyncReturn<Result<``MyStructType``>> as return type in api.rs. Am I missing something or is this not a supported thing?

Generate command says:

> `thread 'main' panicked at 'Failed to parse function output type `SyncReturn<Result<``MyStructType``>>

Thanks.

**Comment by fzyzcjy** [comment]
If it fails, I guess it is not supported yet. However, given the current (new) infra about SyncReturn, it may not be hard to add this. Feel free to PR!
